### PR TITLE
Made alignment changes to audio controls and added missing tile :D

### PR DESCRIPTION
**Pull request by Beta-King**
http://rawgit.com/Beta-King/labyrinth/master/index.html
Fixes issue --> #407

### CODE REVIEW

**Comment by Beta-King** [comment]
please review and merge :D

File: index.html [hunk 1]
```diff
@@ -74,12 +74,12 @@
...
 					<div class="audiosetting">
 						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="previousBackgroundAudio();" title="Previous Audio"><i class="fa fa-step-backward" aria-hidden="true"></i></a>
-						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="nextBackgroundAudio();" title="Next Audio"><i class="fa fa-step-forward" aria-hidden="true"></i></a>
 						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="startBackgroundAudio();" title="Play Audio"><i class="fa fa-play" aria-hidden="true"></i></a>
 						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="stopBackgroundAudio();" title="Stop"><i class="fa fa-pause" aria-hidden="true"></i></a>
+						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="nextBackgroundAudio();" title="Next Audio"><i class="fa fa-step-forward" aria-hidden="true"></i></a>
 					</div>
 					<p class="range-field">
-						<input type="range" onchange="setVolume()" id='volume1' min=0 max=1 step=0.01 value='1'>
+						<input type="range" onchange="setVolume()" id='volume1' min=0 max=100 step=1 value='100'>
```
Comment (duttaditya18) [inline index.html]: Why did you change that? It was for volume. And volume works on from range of 0-1 in js. There was a reason it was put there. Stop doing useless changes.

File: index.html [hunk 2]
```diff
@@ -74,12 +74,12 @@
 				<div class="card-action">
 					<div class="audiosetting">
 						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="previousBackgroundAudio();" title="Previous Audio"><i class="fa fa-step-backward" aria-hidden="true"></i></a>
-						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="nextBackgroundAudio();" title="Next Audio"><i class="fa fa-step-forward" aria-hidden="true"></i></a>
 						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="startBackgroundAudio();" title="Play Audio"><i class="fa fa-play" aria-hidden="true"></i></a>
 						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="stopBackgroundAudio();" title="Stop"><i class="fa fa-pause" aria-hidden="true"></i></a>
+						<a class="waves-effect waves-light btn blue-grey darken-2" onclick="nextBackgroundAudio();" title="Next Audio"><i class="fa fa-step-forward" aria-hidden="true"></i></a>
```
Comment (duttaditya18) [inline index.html]: I think it looked good where it was. No need for this change.

File: index.html (hunk 1 above)
Comment (Beta-King) [reply index.html]: ohkk :D reverting that then :D

File: index.html (hunk 2 above)
Comment (Beta-King) [reply index.html]: its not how you find it generally on other websites hence i changed it :D @duttaditya18

Comment (duttaditya18) [reply index.html]: **It is.** The next button is adjacent to the previous button in most cases.

**Comment by Beta-King** [comment]
Im sorry you find it that way but i just thought it would look more professional if we had the alignment the way i have proposed, have you ever seen another website that has the alignment the way it is currently @duttaditya18 it is just a suggestion to change it...

**Comment by Beta-King** [comment]
Even in the your laptop you can see that the keys are positioned previous - pause/unpause - next

File: index.html (hunk 2 above)
Comment (Beta-King) [reply index.html]: could you please show me that in other renowned websites @duttaditya18

**Comment by Beta-King** [comment]
screen shot 2018-01-11 at 6 16 37 pm --> an example... @duttaditya18

**Comment by Beta-King** [comment]
screen shot 2018-01-11 at 6 19 50 pm --> another example...

**Comment by Beta-King** [comment]
Fixes #408 :D

**Comment by niccokunzmann** [comment]
@duttaditya18 Please do not make "value judgments". https://rfc.zeromq.org/spec:42/C4/ Instead, revert and say why your change is an improvement.
@Beta-King Thanks for your code. Now, we have merge conflicts.
In future, please create separate pull requests for changes which can be done independently. That also allows @duttaditya18 to express concerns through code without breaking your other changes.

**Comment by niccokunzmann** [comment]
@duttaditya18 Thanks for taking the time to review this pull request. I know you are very concerned with the state and quality of the game as you contribute a lot.

**Comment by Beta-King** [comment]
@niccokunzmann --> resolving the conflicts ASAP :D

**Comment by Beta-King** [comment]
Work in progress... :D

**Comment by Beta-King** [comment]
work done :D @niccokunzmann please review and merge :D

**Comment by niccokunzmann** [comment]
Thanks.
